go.mod

```diff
@@ -3,16 +3,27 @@ module github.com/scylladb/gocqlx/v2
 go 1.17
 
 require (
-	github.com/gocql/gocql v0.0.0-20211015133455-b225f9b53fa1
+
+	github.com/gocql/gocql v1.0.0
```

Comment:
Was there a reason the imported version got changed?

Reply:
Done